NorthstarDLL/mods/modmanager.cpp

```diff
@@ -594,6 +609,9 @@ void ModManager::LoadMods()
 
 		Mod mod(modDir, (char*)jsonStringStream.str().c_str());
 
+		if (!mod.m_bEnabled)
+			continue;
+
```

Comment:
What's the purpose of this? I might be mistaken but `mod.m_bEnabled` shouldn't have been touched yet?

Reply:
A leftover from me skipping disabled mods